### ISSUE TITLE
Some clients can't sync after migrate from OC to NC

### ISSUE THREAD

**Issue by tuxmaster** [nextcloud/server#8084]
After update to NC 12.0.4 from OC 10.0.4 an than update to NC 12.0.5 some clients can't sync the files any more. The will only get errors about missing checksums.

The solution for it was:
Clean up the database via:
`UPDATE oc_filecache SET checksum = '' WHERE COALESCE (checksum, '') <> '';`
Then rescan the file system via:
`sudo -u apache scl enable php71 "php occ files:scan --all"`
The original was found [here](https://help.nextcloud.com/t/windows-sync-client-doesnt-sync-all-files-checksum-wrong/14044).
Environment:
OS: CentOS 7.4
PHP: 7.1 via fpm

**Comment by MorrisJobke** [comment]
cc @nickvergessen @rullzer

**Comment by rullzer** [comment]
Ah pfff right.

@nickvergessen probably easiest is to on migration just drop them. Since we handle them not the same as oC for now I think.

**Comment by nickvergessen** [comment]
Yeah fine by me

**Comment by hanscomps** [comment]
Having the same problem.
What is the result of clearing the checksums in the database?
Resync of all files on all clients?  We have TB of data on 100 distributed clients; don't want to disrupt the system too much.

**Comment by dominalien** [comment]
In my specific case (migration from ownCloud to Nextcloud), I got the problem after I installed the Nextcloud client on a machine and after clearing the entries in the database (some 800 of them), it said it was going to redownload some 3000 files. Fortunately, it seems to have downloaded much less in the end.

On the other hand, on a couple of other computers where I left the ownCloud client running, it saw no difference and continued on as if nothing had changed. So, I'm going to just leave the ownCloud client running for everyone else until it either stops working or I know I can install the Nextcloud one without triggering huge downloads (or worse, sync failures) for them.
